### PR TITLE
arm01: Add arm64 utility imagestreams

### PR DESCRIPTION
**Pull request by LorbusChris**

### CODE REVIEW

File: clusters/app.ci/prow/03_deployment/adapter_imagestreams.yaml [hunk 1]
```diff
@@ -53,6 +53,19 @@ items:
       from:
```
Comment (alvaroaleman) [inline clusters/app.ci/prow/03_deployment/adapter_imagestreams.yaml:53]: Can we add them to just the arm cluster? They just occupy space on all the other clusters

Comment (LorbusChris) [reply clusters/app.ci/prow/03_deployment/adapter_imagestreams.yaml:53]: good point. will update.

**Review by alvaroaleman — APPROVED** [review]

**Comment by LorbusChris** [comment]
@alvaroaleman I had to add a newline to make yamllint happy. PTAL

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**